packages/metal-dom/src/keyboardEvents.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+import { object } from 'metal';
+
+export const KEYMAP = {
+  'BACKSPACE': 8,
```

Comment:
Oh and one more thing I've just thought about, we need to export this KEYMAP from the `all/dom.js` file, otherwise people won't be able to access it. I don't think we should export the other functions available here, this object is the one that would be helpful for others.